packages/protocol/scripts/foundry/create_and_migrate_anvil_l2_devchain.sh

```diff
@@ -8,31 +8,14 @@ source $PWD/scripts/foundry/constants.sh
 echo "Generating and running L1 devchain before activating L2..."
 source $PWD/scripts/foundry/create_and_migrate_anvil_devchain.sh
 
-# Stop L1 devchain
-source $PWD/scripts/foundry/stop_anvil.sh
```

Comment:
Are you sure this works? As far as I understand Anvil only writes a state file when it stops, so what you're coping bellow may be from a previous build

Reply:
As far as I can tell, the state file is created during the migration and updated after every state change. In other words, it exists while Anvil is running, and not just when the Anvil process is stopped. 

I guess a good test for this is if the CI devchain publishing step works as expected. I cleaned my `tmp` folder and Foundry artifacts, but didn't run into any issues. I might have missed something, but seems to work as expected.

Reply:
CI seems to pass as expected ref: https://github.com/celo-org/celo-monorepo/actions/runs/10305049035/job/28525795457?pr=11177